script/brave_license_helper.py

```diff
@@ -144,6 +144,8 @@ def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
         'chromium_src',  # Brave's overrides, covered by main notice.
         'node_modules',  # See brave/third_party/npm-* instead.
         '.vscode',       # Automatically added by Visual Studio.
+        'NIST',
```

Comment:
What are these from?

Reply:
will address on Tuesday

Reply:
I added these exceptions for remove the false positive license errors for an unused Rust dep. Since that Rust dep is no longer used, I removed the exceptions here.